<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arithmetic_preWidget</class>
 <widget class="QWidget" name="arithmetic_preWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>arithmetic_preWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_enable">
       <property name="text">
        <string>启用模块</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_output">
       <property name="text">
        <string>输出</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_outputIdx">
       <item>
        <property name="text">
         <string>图像1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图像2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="gb_edit">
     <property name="title">
      <string>编辑</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QSpinBox" name="sp_ROInum">
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="lb_ROInum">
        <property name="text">
         <string>ROI数目</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="bt_select">
        <property name="text">
         <string>选择区域</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_allImgValid">
        <property name="text">
         <string>整幅图片</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cb_img1">
          <item>
           <property name="text">
            <string>图像1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>图像2</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_operator">
          <item>
           <property name="text">
            <string>加</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>减</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>乘</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>除</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>最大</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>最小</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>与</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>或</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>非</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>与非</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>或非</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>异或</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>异或非</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_img2">
          <item>
           <property name="text">
            <string>图像1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>图像2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="4">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lb_a">
          <property name="text">
           <string>参数a</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="dsp_a"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lb_b">
          <property name="text">
           <string>参数b</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="dsp_b"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lb_c">
          <property name="text">
           <string>参数c</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="dsp_c"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
